src/components/containers/visibility.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import device, { size } from 'themes/device'
+import { useBrowserResize } from 'components/hooks/use-browser-resize'
+
+type ResponsiveContainerProps = {
+    children: ReactElement
```

Comment:
I think this should be ReactNode as we may want to pass children in as well when calling it in another component.

Reply:
@sean-binary I was using `reactNode` but as I told you I'm still getting this: 

```
'children' is missing in props validation (eslint react/prop-types)
```

Reply:
Oh right, okay then ignore this comment then.